source/expressions.tex

```diff
@@ -767,9 +767,9 @@
 can be converted to type \tcode{T2}
 if the cv-combined type of \tcode{T1} and \tcode{T2} is \tcode{T2}.
 \begin{note}
-If a program could assign a pointer of type \tcode{T**} to a pointer of
+If a program were able to assign a pointer of type \tcode{T**} to a pointer of
 type \tcode{const} \tcode{T**} (that is, if line \#1 below were
-allowed), a program could inadvertently modify a const object
+allowed), a program would be able to inadvertently modify a const object
```

Comment:
```suggestion
allowed), it would be possible for a const object to be inadvertently modified
```
I think we want to say that it's possible for this to happen, not that a program is able to do it. I still think this is a wording regression, since it creates the impression that this rule prevents inadvertent modification of const objects in general, which it does not.